src/components/App/AppContainer.js

```diff
@@ -1,47 +1,25 @@
 import {connect} from 'react-redux';
 import App from './App';
-import getStore from '../../utils/honestyStore.js';
-import {loadUsers} from './../../utils/slack';
-import {
-	setUsers,
-	setSlackUserFetchError,
-	setCurrentUser,
-	setPrediction
-} from './actions';
-import {
-	setStoreList,
-	setLoadStoreListError,
-	setShowList
-} from './../StoreList/actions';
+import {setSlackUserFetchError, loadUsers} from './actions';
+import {loadStoreList, setShowList} from './../StoreList/actions';
 
 const mapStateToProps = state => {
 	return {
 		storeList: state.storeList,
 		showList: state.showList,
 		slackUserFetchError: state.slackUserFetchError,
-		currentUser: state.currentUser,
 		users: state.users,
 		prediction: state.prediction
 	};
 };
 
 const mapDispatchToProps = dispatch => {
 	return {
-		loadUsers: () =>
-			loadUsers()
-				.then(users => dispatch(setUsers(users)))
-				.catch(() => dispatch(setSlackUserFetchError(true))),
-		setCurrentUser: currentUser => dispatch(setCurrentUser(currentUser)),
-		setPrediction: prediction => dispatch(setPrediction(prediction)),
+		loadUsers,
```

Comment:
If we define `mapDispatchToProps` as on object instead of a function then `bindActionCreators` is used under the hood, letting us use the shorthand for normal actions as well (not just thunks)

E.g.

```
const mapDispatchToProps = {
  loadUSers,
  loadStoreList,
  setShowList, // Shorthand!
  setSlackUserFetchError, // Shorthand!
}
```